packages/core/src/components/external/PlanningView/PlanningView.tsx

```diff
@@ -57,25 +60,46 @@ const PlanningView = ({
   itemList = [],
   changeWeekButton = true,
   returnToDayButton = true,
+  assignedToMeSwitch = true,
```

Comment:
I think default value should be false as it need an extra information in data to work